schematic/__init__.py

```diff
@@ -139,25 +183,6 @@ def request_hook(span: Span, environ: Dict) -> None:
     """
     if not span or not span.is_recording():
         return
-    try:
-        if auth_header := environ.get("HTTP_AUTHORIZATION", None):
-            split_headers = auth_header.split(" ")
-            if len(split_headers) > 1:
-                token = auth_header.split(" ")[1]
-                user_info = info_from_bearer_auth(token)
-                if user_info:
-                    span.set_attribute("user.id", user_info.get("sub"))
-    except Exception:
-        logger.exception("Failed to set user info in span")
-
-    try:
-        if (request := environ.get("werkzeug.request", None)) and isinstance(
-            request, Request
-        ):
-            for arg in request.args:
-                span.set_attribute(key=f"schematic.{arg}", value=request.args[arg])
-    except Exception:
-        logger.exception("Failed to set request info in span")
```

Comment:
I assume we still want this logic to exist? Was it intentionally removed?

Reply:
Thanks for catching this. It was not intentional. Let me add back these lines.